Default to never disable after failure (#91)
Unleash/unleash-client-ruby: README.md

`disable_metrics` | Disables sending metrics to Unleash server. | N | Boolean | `false` |
`custom_http_headers` | Custom headers to send to Unleash. As of Unleash v4.0.0, the `Authorization` header is required. For example: `{'Authorization': '<API token>'}` | N | Hash | {} |
`timeout` | How long to wait for the connection to be established or wait in reading state (open_timeout/read_timeout) | N | Integer | 30 |

> `retry_limit` | How many consecutive failures in connecting to the Unleash server are allowed before giving up. The default here, `Float::INFINITY`, means the client will never give up. | N | Float::INFINITY | 5 |

Reviewer: ```suggestion
`retry_limit` | How many consecutive failures in connecting to the Unleash server are allowed before giving up. The default is to retry indefinitely. | N | Float::INFINITY | 5 |
```
